src/tools/cgexec.c

```diff
@@ -51,28 +55,41 @@ static void usage(int status, const char *program_name)
 	info("change pidlist and children tasks\n");
 #ifdef WITH_SYSTEMD
 	info("  -b				Ignore default systemd delegate hierarchy\n");
+	info("  -r				Replace the systemd scope idle process\n");
```

Comment:
Hmmm.... We should probably be consistent in what we call the idle thread.  (It looks like in the cgcreate man page we call it an "idle process".  FWIW, the actual executable is named `libcgroup_systemd_idle_thread`)

Reply:
Agreed, we need it consistent term. Will try and come up with one.

Reply:
Does `libcgroup_systemd_idle_process` make sense? `thread` vs `process` might be confusing for some.  Let's use `idle process` ?